lib/genesis/pallet.go

```diff
@@ -0,0 +1,161 @@
+package genesis
+
+type contracts struct {
+	CurrentSchedule struct {
```

Comment:
it's a private struct so should the fields inside also be private (or are they needed for encoding/decoding and such)?

Reply:
The struct needs to be public for encoding/decoding